crates/proto/src/version/v662/enums/minecraft_eventing.rs

```diff
@@ -0,0 +1,63 @@
+pub mod MinecraftEventing {
```

Comment:
I don't even know what to say here, this is a true mess and idk what happened

Just maybe remove the extra mod and get each enum a separate file.. I guess